### PR TITLE
Postpone volume decision to end user

### PR DESCRIPTION
**Pull request by cerisier**
As per discussion in #10

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -52,12 +52,10 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local
 # Add PHP-settings
 ADD php-conf.d/ $PHP_INI_DIR/conf.d/
 
-# Create private-files volume, copy sites/default's defaults and make it a volume
+# Create private-files folder, copy sites/default's defaults
 RUN mkdir private && chown -R www-data:www-data /var/www
```
Comment (funkyfuture) [inline Dockerfile]: that can be left out in this case.

Comment (cerisier) [inline Dockerfile]: Not sure I understand.
if we don't explicitly make that folder a volume in the Dockerfile, why leave the comment ?

Comment (funkyfuture) [inline Dockerfile]: i meant the `RUN …`-statement.

(i'm leaving my desk now… into to the sun)

Comment (cerisier) [inline Dockerfile]: Removed it

File: Dockerfile [hunk 2]
```diff
@@ -52,12 +52,8 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local
 # Add PHP-settings
 ADD php-conf.d/ $PHP_INI_DIR/conf.d/
 
-# Create private-files volume, copy sites/default's defaults and make it a volume
-RUN mkdir private && chown -R www-data:www-data /var/www
```
Comment (samos123) [inline Dockerfile]: we still need the  chown -R www-data:www-data /var/www right?

Comment (cerisier) [inline Dockerfile]: You're already doing it there https://github.com/samos123/docker-tutum-drupal/blob/master/entrypoint.sh#L90

Comment (samos123) [inline Dockerfile]: Right, can ignore my comment then :)

**Comment by cerisier** [comment]
Do you need anything else before merging ?

**Comment by moul** [comment]
Looks good to me

**Comment by funkyfuture** [comment]
i'd appreciate some explicit words of wisdom how data persistance would be achieved in the `README.md`

also, can you please adjust the setting of filepermissions in `entrypoint.sh`:
- line 89: cd to `/var/www`
- line 93: cd to `html/sites`

last, but not least, please squash the commits into one, like explained here: https://github.com/samos123/docker-tutum-drupal/pull/3#issuecomment-81680887

**Comment by funkyfuture** [comment]
closed by 5b82ba44b41b7807e663b2ea1c8cd5ccf9a258d1

**Comment by samos123** [comment]
funky please leave the original author in the commit message if you use his changes.

**Comment by funkyfuture** [comment]
i just made a commit from scratch since there was no reaction and it seemed rather urgent to undo my changes.

no harm intended.

**Pull request closed without merging**